pkg/operator/webhookdeployment/webhook.go

```diff
@@ -149,3 +176,16 @@ func (c *csiSnapshotWebhookController) getDeployment(opSpec *operatorapi.Operato
 	return deployment, nil
 
 }
+
+func getWebhookConfig() (*admissionv1.ValidatingWebhookConfiguration, error) {
+	webhookBytes := generated.MustAsset(webhookAsset)
+	requiredObj, err := runtime.Decode(admissionCodecs.UniversalDecoder(admissionv1.SchemeGroupVersion), webhookBytes)
+	if err != nil {
+		return nil, err
+	}
+
+	webhook := requiredObj.(*admissionv1.ValidatingWebhookConfiguration)
```

Comment:
in a perfect world with infinite time we'd move this to library-go's resourceread package :)

Reply:
created https://github.com/openshift/library-go/pull/972